psydac/polar/tests/test_c1_projections.py

```diff
@@ -16,7 +16,7 @@
 
 #==============================================================================
 @pytest.mark.parametrize( 'degrees', [(2,2),(2,3),(3,2),(3,3)] )
-@pytest.mark.parametrize( 'ncells' , [(4,5),(6,6),(11,14)] )
+@pytest.mark.parametrize( 'ncells' , [(8,10),(10,12),(12,14)] )
```

Comment:
the same problem mentioned before, this doesn't work after adding the requirement `Increase the minimum block size for the cart decomposition to be p+1 in each direction instead of p`